pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -0,0 +1,391 @@
+        this.bkPort = bkPort;
+    }
+
+    public void setBkDir(String bkDir) {
+        this.bkDir = bkDir;
+    }
+
+    public void setAdvertisedAddress(String advertisedAddress) {
+        this.advertisedAddress = advertisedAddress;
+    }
+
+    public void setConfig(ServiceConfiguration config) {
+
+        this.config = config;
+    }
+
+    public void setFnWorkerService(WorkerService fnWorkerService) {
+        this.fnWorkerService = fnWorkerService;
+    }
+
+    public void setConfigFile(String configFile) {
+        this.configFile = configFile;
+    }
+
+    public void setWipeData(boolean wipeData) {
+        this.wipeData = wipeData;
+    }
+
+    public void setNumOfBk(int numOfBk) {
+        this.numOfBk = numOfBk;
+    }
+
+    public void setZkPort(int zkPort) {
+        this.zkPort = zkPort;
+    }
+
+    public void setZkDir(String zkDir) {
+        this.zkDir = zkDir;
+    }
+
+    public void setNoBroker(boolean noBroker) {
+        this.noBroker = noBroker;
+    }
+
+    public void setOnlyBroker(boolean onlyBroker) {
+        this.onlyBroker = onlyBroker;
+    }
+
+    public void setNoFunctionsWorker(boolean noFunctionsWorker) {
+        this.noFunctionsWorker = noFunctionsWorker;
+    }
+
+    public void setFnWorkerConfigFile(String fnWorkerConfigFile) {
+        this.fnWorkerConfigFile = fnWorkerConfigFile;
+    }
+
+    public void setNoStreamStorage(boolean noStreamStorage) {
+        this.noStreamStorage = noStreamStorage;
+    }
+
+    public void setStreamStoragePort(int streamStoragePort) {
+        this.streamStoragePort = streamStoragePort;
+    }
+
+    public void setHelp(boolean help) {
+        this.help = help;
+    }
+
+    public ServiceConfiguration getConfig() {
+        return config;
+    }
+
+    public String getConfigFile() {
+        return configFile;
+    }
+
+    public boolean isWipeData() {
+        return wipeData;
+    }
+
+    public int getNumOfBk() {
+        return numOfBk;
+    }
+
+    public int getZkPort() {
+        return zkPort;
+    }
+
+    public int getBkPort() {
+        return bkPort;
+    }
+
+    public String getZkDir() {
+        return zkDir;
+    }
+
+    public String getBkDir() {
+        return bkDir;
+    }
+
+    public boolean isNoBroker() {
+        return noBroker;
+    }
+
+    public boolean isOnlyBroker() {
+        return onlyBroker;
+    }
+
+    public boolean isNoFunctionsWorker() {
+        return noFunctionsWorker;
+    }
+
+    public String getFnWorkerConfigFile() {
+        return fnWorkerConfigFile;
+    }
+
+    public boolean isNoStreamStorage() {
+        return noStreamStorage;
+    }
+
+    public int getStreamStoragePort() {
+        return streamStoragePort;
+    }
+
+    public String getAdvertisedAddress() {
+        return advertisedAddress;
+    }
+
+    public boolean isHelp() {
+        return help;
+    }
+
+    @Parameter(names = { "-c", "--config" }, description = "Configuration file path", required = true)
```

Comment:
These are from JCommander. Maybe it doesn't apply when not running from CLI

Reply:
Don't know what be done right now, I haven't changed the annotations in this pr.